tests/steamship_tests/plugin/integration/test_e2e_async_blockifier.py

```diff
@@ -53,4 +46,3 @@ def test_e2e_async_blockifier_plugin():
                 for tag in block.tags:
                     assert tag.name is not None
                     assert tag.kind is not None
-            file.delete()
```

Comment:
Did you mean to delete this delete?

Reply:
Nevermind you did, because it's within a resource cleanup block

Reply:
Yes, we now can't execute `delete()` twice without raising an Exception which I think is what we want.